src/decoder/Makefile

```diff
@@ -7,6 +7,7 @@ TESTFILES =
 
 OBJFILES = training-graph-compiler.o lattice-simple-decoder.o lattice-faster-decoder.o \
    lattice-faster-online-decoder.o simple-decoder.o faster-decoder.o \
+	 lattice-incremental-decoder.o \
```

Comment:
again, odd whitespace.  These should be spaces not tabs.  Maybe you have your editor set up wrong.